src/System.CommandLine.Tests/OptionTests.cs

```diff
@@ -240,7 +240,8 @@ public void Argument_takes_option_alias_as_its_name_when_it_is_not_provided()
         {
             var command = new Option("--alias", arity: ArgumentArity.ZeroOrOne);
 
-            command.ArgumentHelpName.Should().Be("alias");
+            command.Name.Should().Be("alias");
+            command.ArgumentHelpName.Should().Be(null);
```

Comment:
We can leave off the `ArgumentHelpName` assertion as it's unrelated to name of the test.